packages/http-client-python/emitter/src/lib.ts

```diff
@@ -61,12 +112,6 @@ const libDef = {
         default: paramMessage`Can't generate Python client code from this TypeSpec. Please open an issue on https://github.com/microsoft/typespec'.${"stack"}`,
       },
     },
-    "invalid-models-mode": {
-      severity: "error",
-      messages: {
-        default: paramMessage`Invalid value '${"inValidValue"}' for 'models-mode' of tspconfig.yaml and expected values are 'dpg'/'none'.`,
-      },
-    },
```

Comment:
Maybe we could change severity from "error" to "warning" instead of deleting it.